### PR TITLE
Renames calibers

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Renames calibers, most of these are real, some of them are ~~memes~~ more questionable. I've avoided giving rifles super common IRL rounds on purpose. Pistol rounds are unlikely to change, but Bay numbers are annoying to me and less noticeable.

RIFLES

Standard Rifle - .300BL
Military rifle- 6.5mm
RIFLE LARGE - .338L
ANTIMAT - 12.7mm

PISTOLS

PISTOL - 9MM
PISTOL SMALL - .22LR
ANTIQUE - 45 (since they don't make a 46)
MAGNUM - .44
MAGNUM LARGE - .454, removed the mm.

## Why It's Good For The Game
Makes the gun calibers more recognizeable, atleast to me than something like..

7mm, and then there's a 7mmr???

### CODE REVIEW

**Comment by BurpleBineapple** [comment]
Seems like an arbitrary change that's just going to confuse people for a few days. I don't really care either way but it doesn't really feel like this is needed.

File: code/modules/boh/makeshift_weapons/improv_rifle.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	force = 10
 	slot_flags = SLOT_BACK
 	origin_tech = list(TECH_COMBAT=2, TECH_MATERIAL=1, TECH_ESOTERIC=1)
-	caliber = "7mmR"
+	caliber = ".22LRR"
```
Comment (BurpleBineapple) [inline code/modules/boh/makeshift_weapons/improv_rifle.dm:17]: also what. .22LR? 7mmR would be a 7.62mm rimmed cartridge, AKA, the mosin nagant's bullet.

File: code/modules/fabrication/designs/general/designs_arms_ammo.dm [hunk 2]
```diff
@@ -41,107 +41,107 @@
...
 	path = /obj/item/ammo_magazine/mil_rifle/sec/large
 
 /datum/fabricator_recipe/arms_ammo/hidden/bullpupbad_practice
-	name = "rifle magazine (5mmR), (practice)"
+	name = "rifle magazine (6.5mm), (practice)"
 	path = /obj/item/ammo_magazine/mil_rifle/practice
 
 /datum/fabricator_recipe/arms_ammo/hidden/magazine_arifle
-	name = "rifle magazine (7mmR)"
+	name = "rifle magazine (.300BL)"
```
Comment (BurpleBineapple) [inline code/modules/fabrication/designs/general/designs_arms_ammo.dm:144]: Your calibers aren't consistent. Here you've named 7mmR .300BLK

File: code/modules/fabrication/designs/general/designs_arms_ammo.dm [hunk 3]
```diff
@@ -153,7 +153,7 @@
 	path = /obj/item/ammo_magazine/skrell_shotgun
 
 /datum/fabricator_recipe/arms_ammo/hidden/lmg
-	name = "lmg box magazine (7mmR)"
+	name = "lmg box magazine (.338L)"
```
Comment (BurpleBineapple) [inline code/modules/fabrication/designs/general/designs_arms_ammo.dm:156]: and here you've named the same caliber .338L?

File: code/modules/research/designs/designs_mechfab.dm [hunk 4]
```diff
@@ -567,13 +567,13 @@
 	materials = list(MATERIAL_STEEL = 7000)
 
 /datum/design/item/mechfab/mech_ammo/ar
-	name = "large 7mmR ammo box"
+	name = "large 6.5mm ammo box"
```
Comment (BurpleBineapple) [inline code/modules/research/designs/designs_mechfab.dm:570]: And then here you've named it 6.5mm?

File: code/modules/research/designs/designs_mechfab.dm [hunk 5]
```diff
@@ -567,13 +567,13 @@
...
 	id = "mech_ar_ammo"
 	build_path = /obj/item/ammo_magazine/mech/rifle
 	materials = list(MATERIAL_STEEL = 9000)
 
 /datum/design/item/mechfab/mech_ammo/har
-	name = "large 5mm ammo box"
+	name = "large .338L ammo box"
```
Comment (BurpleBineapple) [inline code/modules/research/designs/designs_mechfab.dm:576]: And - then you've named 5mm to .338L. None of these make sense.

**Review by BurpleBineapple — CHANGES_REQUESTED** [review]

**Comment by Anton-Kr** [comment]
Honestly I am a bit fundamentally against this change since I believe it will simply confuse people. I get the gun nut want to make the calibers make more sense. I wouldn't mind some form of edits to the calibers, but I'd prefer with stick to a single metric convention (ie 10mm 7mmR) without huge modifications to what the players are used to. I'd quite like some BETTER distinction for 7mm vs 7mmR round because I find that confusing, and perhaps a metric conversion of the .454. I understand that this is quite contrary to the change presented, but that's really the only way I see myself merging this PR in good faith.

**Review by Anton-Kr — CHANGES_REQUESTED** [review]
Please fix issues and make calibers aligned with current code standards. Please avoid modifying commonly used calibers.

**Pull request closed without merging**